sycl/test/check_device_code/id_queries_fit_int.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -fsycl -fsycl-id-queries-fit-in-int -O0 -c -S -emit-llvm -o %t.ll %s
+// RUN: %clangxx -fsycl -Xclang -fsycl-is-host -fsycl-id-queries-fit-in-int -fno-inline-functions -O1 -c -S -emit-llvm -o %t.ll %s
```

Comment:
1. Why do you pass `-fsycl-is-host` bypassing the driver? As you can see by pre-commit test results it's not valid!
2. Please, clarify why `-fno-inline-functions` is needed?
3. Why `-O0` changed to `-O1`?
4. What is wrong with existing test?